### PR TITLE
Fixes disposal bins/outlets breaking the floor tiles when things come out of them, other minor fixes

### LINKED ISSUE #7440: Disposals bins still explode the floor plating when receiving an item (calling ex_act for some reason?)
I don't know what the fuck this is but I'm pretty sure it should NOT be happening. Been going on for as long as I remember.

### PR DESCRIPTION
**Pull request by Sprok0**
Mail deliveries using the disposal system won't break the floor tiles anymore.
You can't open the nonfunctional disposal bin menu on disposal chutes.
Fixes disposal bins appearing as a normal disposal pipe for part of their deconstruction.
Removes a bunch of relative pathing.

Fixes  #7440

### CODE REVIEW

File: code/modules/recycling/disposal.dm [hunk 1]
```diff
@@ -1446,18 +1448,18 @@
...
 	desc = "A broken piece of disposal pipe."
 
-	New()
-		..()
-		update()
-		return
+/obj/structure/disposalpipe/broken/New()
+	..()
+	update()
+	return
```
Comment (Intigracy) [inline code/modules/recycling/disposal.dm]: remove this return

File: code/modules/recycling/disposal.dm [hunk 2]
```diff
@@ -1476,90 +1478,91 @@
...
-		if(H)
-			for(var/atom/movable/AM in H)
-				AM.forceMove(src.loc)
-				AM.pipe_eject(dir)
-				spawn(5)
-					if(AM)
-						AM.throw_at(target, 3, 1)
-			H.vent_gas(src.loc)
-			qdel(H)
+	return
```
Comment (Intigracy) [inline code/modules/recycling/disposal.dm]: And this one

File: code/modules/recycling/disposal-construction.dm [hunk 3]
```diff
@@ -20,260 +20,260 @@
...
+			return /obj/machinery/disposal
+		if(7)
+			return /obj/structure/disposaloutlet
+		if(8)
+			return /obj/machinery/disposal/deliveryChute
+		if(9,10)
+			return /obj/structure/disposalpipe/sortjunction
+		if(11, 12)
+			return /obj/structure/disposalpipe/wrapsortjunction
+	return
```
Comment (Intigracy) [inline code/modules/recycling/disposal-construction.dm]: and this one

**Review by Intigracy — CHANGES_REQUESTED** [review]
trailing returns

**Review by Intigracy — APPROVED** [review]

**Comment by gbasood** [comment]
and here it's been around for so long I thought it was intended

**Comment by clusterfack** [comment]
I literally remember someone else making a PR to fix this, what happened

**Comment by ghost** [comment]
What about if you send sand through the delivery pipes on the asteroid, does this fix the sand blasting the wall and causing a tonne of lag for anyone around? I should probably have reported this as a bug ages ago.

**Comment by Sprok0** [comment]
I haven't heard of that issue, but I don't think this would do anything about it

**Pull request merged**